src/ui-presets/img.js

```diff
@@ -54,7 +53,6 @@ class ImgUI extends Component {
       <div className={style.playbackGuiWrapper}>
         <PlayerArea name={'PresetArea'}>
           <div className={style.playerGui} id="player-gui">
-            <OverlayAction />
```

Comment:
no, this changes is effects only non durational image (which can never be part of a playlist) since it is in img preset, images playlist is played using the playback preset